cmake/GrpcTargets.cmake

```diff
@@ -63,16 +63,20 @@ function(generate_grpc_files)
       set(path_base "${name_base}")
     endif()
 
+    # resolve root_path, proto_file to real path's - protoc check that root_path is prefix of proto_file (this can be non true if project inside folder sym linked to other dir)
+    get_filename_component(real_root_path ${root_path} REALPATH)
```

Comment:
Please move changes in this file into a separate PR. They could be merged faster and as separate fix